exporters/otlp/options.go

```diff
@@ -43,3 +43,22 @@ func WithMetricExportKindSelector(selector metricsdk.ExportKindSelector) Exporte
 		cfg.exportKindSelector = selector
 	}
 }
+
+// ProtocolDriverOption provides options for setting up a split driver.
+type SplitDriverOption func(d *splitDriver)
```

Comment:
This doesn't match our configuration style guide: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#option

If we are going to switch to an options approach we should follow the other patterns in the repo.